Modules/IO/IPL/include/itkIPLFileNameList.h

```diff
@@ -93,21 +93,14 @@ class IPLFileSortInfo
 
   virtual ~IPLFileSortInfo();
 
-  IPLSetMacroDeclaration(ImageFileName, std::string);
-  IPLGetMacroDeclaration(ImageFileName, std::string);
-  IPLSetMacroDeclaration(SliceLocation, float);
-  IPLGetMacroDeclaration(SliceLocation, float);
-  IPLSetMacroDeclaration(SliceOffset, int);
-  IPLGetMacroDeclaration(SliceOffset, int);
-  IPLSetMacroDeclaration(EchoNumber, int);
-  IPLGetMacroDeclaration(EchoNumber, int);
-  IPLSetMacroDeclaration(ImageNumber, int);
-  IPLGetMacroDeclaration(ImageNumber, int);
-  IPLSetMacroDeclaration(Data, void *);
-  IPLGetMacroDeclaration(Data, const void *);
+  IPLSetMacroDeclaration(ImageFileName, std::string) IPLGetMacroDeclaration(ImageFileName, std::string)
+    IPLSetMacroDeclaration(SliceLocation, float) IPLGetMacroDeclaration(SliceLocation, float)
+      IPLSetMacroDeclaration(SliceOffset, int) IPLGetMacroDeclaration(SliceOffset, int)
+        IPLSetMacroDeclaration(EchoNumber, int) IPLGetMacroDeclaration(EchoNumber, int)
+          IPLSetMacroDeclaration(ImageNumber, int) IPLGetMacroDeclaration(ImageNumber, int)
+            IPLSetMacroDeclaration(Data, void *) IPLGetMacroDeclaration(Data, const void *)
```

Comment:
clang-format screwed this up, you need to add this macro to that list in the .clang-format file to get it to format correctly.

Reply:
Yes, sorry about that. I fixed the file.